functions.inc.php

```diff
@@ -345,7 +345,7 @@ function callrecording_get_config($engine) {
 		$exten = 's';
 
 		$ext->add($context, $exten, '', new ext_set('ONETOUCH_REC_SCRIPT_STATUS', ''));
-		$ext->add($context, $exten, '', new ext_system(FreePBX::Config()->get('ASTVARLIBDIR').'/bin/one_touch_record.php "${CHANNEL(name)}"'));
+		$ext->add($context, $exten, '', new ext_system('${AMPBIN}/one_touch_record.php "${CHANNEL(name)}"'));
```

Comment:
AMPBIN is never defined in this context. Unreliable call. Not sure what it would be or where it would come from.

The code should be:
```
FreePBX::Config()->get('AMPBIN')
```